tests/integration/test_authd/data/test_cases/cases_authd_use_password_invalid.yaml

```diff
@@ -0,0 +1,9 @@
+- name: Use empty password file.
+  description: Set the use_password tag with the value 'yes',
+               create the file authd.pass and keep it empty
+               making authd impossible to start.
+  configuration_parameters:
+    USE_PASSWORD: 'yes'
+  metadata:
+    error: Empty password provided.
+    password: empty
```

Comment:
This case is when nothing is added to it, but what happens if we put spaces as a value? Does it work?

Reply:
Case added in commit [`c817b8b`](https://github.com/wazuh/wazuh-qa/pull/3721/commits/c817b8bd7907fffe692553395804b7a2ec23518e), it is now expected to fail until the issue wazuh/wazuh#16282 is resolved